js_modules/dagster-ui/packages/ui-components/src/theme/color.tsx

```diff
@@ -8,6 +8,29 @@ const getColor = memoize((semanticName: ColorName): string => {
   return palette[semanticName];
 });
 
+export function replaceAlpha(rgbaString: string, newAlpha: number) {
+  // Check if the input string is in the correct format
+  const rgbaRegex = /^rgba?\((\s*\d+\s*),(\s*\d+\s*),(\s*\d+\s*),(\s*[\d.]+\s*)\)$/;
+  const match = rgbaString.match(rgbaRegex);
+  if (!match) {
```

Comment:
Would be nice to also require match.length === 3 or 4 here, just in case since it's required on lines 21-23

Reply:
the regex requires that by the number of matches in it